src/javascript/ContentManager/ContentManager.utils.js

```diff
@@ -59,6 +59,36 @@ function ellipsizeText(text, maxLength) {
     return ellipsize(text, maxLength || 100, {chars: [' ', '&']});
 }
 
+function getZipName(name) {
+    if (name.lastIndexOf('.') > 0) {
+        return name.substr(0, name.lastIndexOf('.')) + '.zip';
+    }
+
+    return name + '.zip';
+}
+
+function getNameWithoutExtension(name) {
+    if (name.lastIndexOf('.') > 0) {
+        return name.substr(0, name.lastIndexOf('.'));
+    }
+
+    return name;
+}
+
+function getNewCounter(nodes) {
+    let max = 0;
+    nodes.forEach(node => {
+        let counter = node.name.match(/[0-9]+.zip/g);
```

Comment:
You could probably make this method more generic, by replacing this line by some code removing the file extension (whatever it is) from the name (ie. calling the method above).